src/TitleDetails/TitleDetails.js

```diff
@@ -83,6 +88,11 @@ const TitleDetails = ({
   const { id: poLineId, receiptDate, poLineNumber, checkinItems } = poLine;
   const titleId = title.id;
   const isOrderClosed = order.workflowStatus === ORDER_STATUSES.closed;
+  const pieceLocationId = pieceValues.locationId;
+  const poLineLocationIds = useMemo(() => poLine?.locations?.map(({ locationId }) => locationId), [poLine]);
```

Comment:
leverage complex data paths to separate var above. `const something = poLine?.locations`,